docs/dxp/latest/en/developing-applications/developing-a-java-web-application/using-mvc/tag-libraries/clay-tag-library/clay-badges.md

```diff
@@ -0,0 +1,87 @@
+# Clay Badges
+
+Badges help highlight important information such as notifications or new and unread messages. Badges have circular borders and are only used to specify a number. This covers the different types of Clay badges you can add to your app.
+
+## Badge Types
+
+The following badge styles are available:
+
+Primary badge:
+
+```markup
+<div class="col-md-1">
+    <clay:badge label="8" />
+
+    <div>Primary</div>
+</div>
+```
+
+![A primary badge is bright blue, commanding attention like the primary button of a form.](./clay-badges/images/01.png)
+
+Secondary badge:
+
+```markup
+<div class="col-md-1">
+    <clay:badge label="87" style="secondary" />
+
+    <div>Secondary</div>
+</div>
+```
+
+![A secondary badge is light-grey and draws less focus than a primary button.](./clay-badges/images/02.png)
+
+Info badge:
+
+```markup
+<div class="col-md-1">
+    <clay:badge label="91" style="info" />
+
+    <div>Info</div>
+</div>
+```
+
+![A info badge is dark blue and meant for numbers related to general information.](./03-clay-badges/images/03.png)
```

Comment:
This image link is bad. There are some other image links that need checking too.